features/bevragen/persoon/verblijfplaats/buitenland/protocollering-antwoord.feature

```diff
@@ -0,0 +1,37 @@
+#language: nl
+
+Functionaliteit: protocollering van het antwoord voor verblijfplaats buitenland
+
```

Comment:
Zie opmerking in regel 4 van features/bevragen/persoon/verblijfplaats/onbekend/protocollering-antwoord.feature.